README.md

```diff
@@ -61,14 +69,34 @@ There are currently the following workflows available:
         - Remove `if: false` line;
         - Configure Secrets for [Signing Properties](https://github.com/r0adkll/sign-android-release);
         - Configure Secrets for [Firebase distribution](https://github.com/wzieba/Firebase-Distribution-Github-Action);
-- [Publish Debug](.github/workflows/publish_debug.yaml)
+- [Publish Debug to firebase](.github/workflows/publish_debug.yaml)
     - Will build debug on PR from the `release/*` branch;
     - Prepare release notes upon commits starting from the main branch;
     - Publish build to the Firebase Distribution;
     - To use:
         - Remove `if: false` line;
         - Configure Secrets for [Signing Properties](https://github.com/r0adkll/sign-android-release);
         - Configure Secrets for [Firebase distribution](https://github.com/wzieba/Firebase-Distribution-Github-Action);
+- [Publish Debug to App Center](.github/workflows/beta-to-testers-app-center.yaml)
+    - Will build debug from main branch;
+    - Prepare release notes upon commits starting from the last tag;
+    - Publish build to the app center;
+    - To use:
+        - Remove `if: false` line;
+        - Create project on the App Center and replace name stubs
+        - Create group on the App Center and replace group name stub
+        - Configure Secrets for [Signing Properties](https://github.com/r0adkll/sign-android-release);
+        - Configure Secrets for [Appcenter distribution](https://github.com/wzieba/AppCenter-Github-Action);
+- [Publish Release to App Center](.github/workflows/release-to-internal-google-play.yaml)
+    - Will build release from created tag;
+    - Publish build to the google play internal testing channel;
+    - To use:
+        - Remove `if: false` line;
+        - Create project on the App Center and replace name stubs
+        - Configure Secrets for [Signing Properties](https://github.com/r0adkll/sign-android-release);
+        - Generate service key on google play (admin role required) [Google Play service key](https://corewillsoft.atlassian.net/wiki/spaces/XMP2GO/pages/556367875/Retrieve+google+play+API+service+account+json+and+grant+access);
```

Comment:
this is not xmp2go project. Please, double check all the references -> let's keep the template project-agnostic

Reply:
This is general documentation. I can move to the general company confluence and attach it here. What do you think?